<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMROEditor</class>
 <widget class="QDialog" name="IMROEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMEC Readout Table Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>292</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>61</number>
   </attribute>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Bank</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Refid</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>APgain</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LFgain</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="statusLbl">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>71</red>
         <green>3</green>
         <blue>34</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>71</red>
         <green>3</green>
         <blue>34</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBut">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>131</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Bank</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Refid</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="refidBut">
      <property name="text">
       <string>Set All</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="refidSB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="bankSB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="apBut">
      <property name="text">
       <string>Set All</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>LFgain</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="lfCB">
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>125</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>250</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>500</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1500</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2500</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="bankBut">
      <property name="text">
       <string>Set All</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>APgain</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="apCB">
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>125</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>250</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>500</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1500</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2500</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="lfBut">
      <property name="text">
       <string>Set All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="prbHdr">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Probe option:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="prbLbl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>71</red>
            <green>3</green>
            <blue>34</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>71</red>
            <green>3</green>
            <blue>34</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>106</red>
            <green>104</green>
            <blue>100</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="defaultBut">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>defaultBut</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>loadBut</tabstop>
  <tabstop>saveBut</tabstop>
  <tabstop>bankSB</tabstop>
  <tabstop>bankBut</tabstop>
  <tabstop>refidSB</tabstop>
  <tabstop>refidBut</tabstop>
  <tabstop>apCB</tabstop>
  <tabstop>apBut</tabstop>
  <tabstop>lfCB</tabstop>
  <tabstop>lfBut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
